examples/controlnet/requirements.txt

```diff
@@ -1,4 +1,4 @@
-accelerate==0.19.0
+git+https://github.com/huggingface/accelerate
```

Comment:
Just curious. Why change to the git version of `accelerate`? Is there any new feature needed that are not available in 0.19.0?

Reply:
I encountered an error when using accelerate 0.19.0 with diffusers' models, even though it worked fine before. The log:
`Traceback (most recent call last):
  File "train_controlnet_mlflow.py", line 24, in <module>
    import accelerate
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/__init__.py", line 3, in <module>
    from .accelerator import Accelerator
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/accelerator.py", line 33, in <module>
    from .checkpointing import load_accelerator_state, load_custom_state, save_accelerator_state, save_custom_state
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/checkpointing.py", line 24, in <module>
    from .utils import (
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/utils/__init__.py", line 119, in <module>
    from .megatron_lm import (
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/accelerate/utils/megatron_lm.py", line 32, in <module>
    from transformers.modeling_outputs import (
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/__init__.py", line 26, in <module>
    from . import dependency_versions_check
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/dependency_versions_check.py", line 57, in <module>
    require_version_core(deps[pkg])
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/utils/versions.py", line 117, in require_version_core
    return require_version(requirement, hint)
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/utils/versions.py", line 111, in require_version
    _compare_versions(op, got_ver, want_ver, requirement, pkg, hint)
  File "/nas/.conda/envs/controlnet/lib/python3.8/site-packages/transformers/utils/versions.py", line 44, in _compare_versions
    raise ImportError(
ImportError: accelerate>=0.20.3 is required for a normal functioning of this module, but found accelerate==0.19.0.
Try: pip install transformers -U or pip install -e '.[dev]' if you're working with git main`

Reply:
Looks like `accelerate>=0.20.3` is a requirements from `transformers`. Can you find out which `transformers` version demand this? A simple `pip freeze | grep transformers` in your current env would suffice.

P/s: There is actually no issue with using the latest git version of a library. But since `transformers`, as a project, moves really fast, I believe it is beneficial for us to be aware of its requirements